calculator.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace calculator
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Create a Calculator application, with four methods to perform add, diff, mult and div.
+            //Each method will accept two double inputs and return the result double value.
+          
+            bool flag=true;
+            do { 
+                Console.WriteLine("Enter the First digit");
+            int firstNumber = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter the secound number");
+            int secoundNumber = int.Parse(Console.ReadLine());
+                OperationsCalc calculator = new OperationsCalc(firstNumber,secoundNumber);
+            Console.WriteLine("Enter the operation to perform");
+            char Operator = char.Parse(Console.ReadLine());
+            
+                switch (Operator)
+                {
+                    case '+':
+                        Console.WriteLine(calculator.sum1);
+                        break;
+                    case '-':
+                        Console.WriteLine(calculator.subtract);
+                        break;
+                    case '*':
+                        Console.WriteLine(calculator.multiply);
+                        break;
+                    case '/':
+                        Console.WriteLine(calculator.divide);
+                        break;
+                    
+
+
+                }
+                Console.WriteLine("Do you want to continue Y/N");
+                char option=char.Parse(Console.ReadLine());
+                if (option == 'Y') {  flag = true; }
+                else { flag = false; }
```

Comment:
instead of if else, this can be done in a single line

flag = (option == 'Y');

Reply:
Changes made